service/ui.go

```diff
@@ -92,17 +91,34 @@ func NewUIServer(settings *evergreen.Settings, queue amboy.Queue, home string, f
 			settings.Jira.Password),
 	}
 
+	plugins := plugin.GetPublished()
 	uis.PanelManager = &plugin.SimplePanelManager{}
-	if err := uis.PanelManager.RegisterPlugins(plugin.UIPlugins); err != nil {
+
+	if err := uis.PanelManager.RegisterPlugins(plugins); err != nil {
 		return nil, errors.Wrap(err, "problem initializing plugins")
 	}
 
+	catcher := grip.NewBasicCatcher()
+	for _, pl := range plugins {
+		// get the settings
+		pluginSettings, ok := uis.Settings.Plugins[pl.Name()]
+		if !ok {
```

Comment:
This validation requires that every possible plugin be defined in the settings. I think it should continue if not found